unified-scheduler-pool/src/lib.rs

```diff
@@ -577,8 +966,10 @@ mod tests {
 
     #[derive(Debug)]
     struct AsyncScheduler<const TRIGGER_RACE_CONDITION: bool>(
-        PooledScheduler<DefaultTaskHandler>,
```

Comment:
now stop using the impl by wrapping as the impl is getting fleshed out, meaning it's harder to use for a race condition test